.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
+}
```

Comment:
@RomanGotsiy Actually, can you also remove the `dist` npm script? This is now obsolete since I use `babel-it`. I use `babel-it` on my machine, when I want to publish the new versions on npm. 😁